### DISCUSSION TITLE
多语言相关

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#1481] [提问] [answered]
刚刚想到了所以说就来发发
或许龙猫可以加个本地化键名，在一个目录里单独列出语言文件，直接写死好像并不是很方便（
（例如要改什么显示文字的东西还要去改代码（）
同样可以读取位于目录自定义的语言文件，对于其他语言的翻译工作实际上可以给社区进行处理，如果有例如UI冲突的可以单独列出尝试整改，可以作为一个本地化测试的功能看起来还是不错的（

**Comment by Chiloven945** [comment]
还是解释一下，我这里说的多语言是作为测试作用的，如果可行的话就可以考虑一下实装看看（

**Comment by MoYuan-CN** [comment]
如果有想法,可提交 [新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new/choose)

**Reply by Chiloven945** [reply]
鉴于前车之鉴我才来交的discussion

**Comment by so1ve** [comment]
#119 
一个是拖慢开发进度，一个是需要重新设计UI

**Reply by Chiloven945** [reply]
开发进度的确有所影响，但是我说的是对多语言功能交由社区进行测试而不是实装，等到会出现UI问题可以反馈并进行修复

**Reply by so1ve** [reply]
啊，合理的
但是龙猫应该不会自己去写这个功能……如果有兴趣欢迎PR

**Reply by Chiloven945** [reply]
我不会.net所以说我也写不了（

**Comment by WForst-Breeze** [comment, marked as answer]
多语言暂时不会作支持，尽管翻译工作可以交给社区完成，但是 UI 的处理仍旧是一件相对棘手的事情，目前 PCL2 需要首先进行其核心功能和中心工作的制作和其它事项和功能的调试，因此这会极大拖慢开发进度，这是其一；其二，PCL2 目前受众几乎仅针对中国大陆地区用户，在支持多语言后，下载源国际适配和其他问题仍然需要进行协商和处理，因此该功能优先级过低，可能暂不会处理。

**Reply by Chiloven945** [reply]
嗯的确是一个问题，我就是说想先试试看效果怎么样(
